### PR TITLE
added new FVM exception

### PR DESCRIPTION
**Pull request by laurens-ritense**
### Describe the changes
Link to the related Github issue: 

**Relevant comments:**
> 

### Breaking changes

- [x] The contribution only contains changes that are not breaking.

### Documentation

- [ ] Release notes have been written for these changes. 

Link to the pull request in the [Valtimo documentation repository](https://github.com/valtimo-platform/valtimo-documentation):
>

New features or changes that have been introduced have been documented.
- [ ] Yes
- [ ] Not applicable

### Tests
Unit tests have been added that cover these changes
- [ ] Yes
- [x] Not applicable

Integration tests have been added that cover these changes
- [ ] Yes
- [x] Not applicable

### Security
The [Secure by Default principle](https://en.wikipedia.org/wiki/Secure_by_default) has been applied to these changes
- [ ] Yes
- [x] Not applicable

Added or changed REST API endpoints have authentication and authorization in place
- [ ] Yes
- [x] Not applicable

Valtimo access control checks have been implemented
- [ ] Yes
- [x] Not applicable

### Dependencies
Newly added dependencies do not introduce known vulnerabilities/CVE's and are in line with the Valtimo license
- [ ] Yes
- [ ] Not applicable

### CODE REVIEW

**Comment by laurens-ritense** [comment]
Must be merged together with https://github.com/valtimo-platform/valtimo-frontend-libraries/pull/1264

File: form-view-model/src/main/kotlin/com/ritense/formviewmodel/web/rest/dto/SingleFormError.kt [hunk 1]
```diff
@@ -16,15 +16,7 @@
...
-
-    val error: String = error ?: UNKNOWN_FORM_ERROR
-
-    companion object {
-        const val UNKNOWN_FORM_ERROR = "Unknown Form Error"
-    }
-
-}
\ No newline at end of file
+class SingleFormError (
```
Comment (theo-ritense) [inline form-view-model/src/main/kotlin/com/ritense/formviewmodel/web/rest/dto/SingleFormError.kt:19]: This is technically a breaking change

File: form-view-model/src/main/kotlin/com/ritense/formviewmodel/error/MultipleFormException.kt [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" basis,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package com.ritense.formviewmodel.error
+
+class MultipleFormException(
```
Comment (theo-ritense) [inline form-view-model/src/main/kotlin/com/ritense/formviewmodel/error/MultipleFormException.kt]: This name kind of implies an exception about multiple forms. However, it's about one form having multiple errors.
Something in the line of `FormValidationException` would be better imho.

File: form-view-model/src/main/kotlin/com/ritense/formviewmodel/web/rest/dto/MultipleFormError.kt [hunk 3]
```diff
@@ -16,14 +16,8 @@
...
 
-    var error: String = error ?: UNKNOWN_BUSINESS_RULE_ERROR
-
-    companion object {
-        const val UNKNOWN_BUSINESS_RULE_ERROR = "Unknown Business Rule Error"
-    }
-
-}
\ No newline at end of file
+class MultipleFormError (
```
Comment (theo-ritense) [inline form-view-model/src/main/kotlin/com/ritense/formviewmodel/web/rest/dto/MultipleFormError.kt]: This is technically a breaking change.

**Review by ivo-ritense — DISMISSED** [review]

**Review by ivo-ritense — DISMISSED** [review]

**Review by ivo-ritense — APPROVED** [review]

**Pull request merged**